roles/kube_nginx_ingress/tasks/main.yml

```diff
@@ -1,16 +1,29 @@
+- name: 'create tempfile for nginx-ingress values'
+  tempfile:
+    state: file
+    suffix: yml
+  register: nginx_ingress_value_file
+
+- debug:
+    var: nginx_ingress_value_file.path
+  when: debug|bool
+
+- name: 'copy nginx-ingress values into temporary file'
+  copy:
+    src: nginx_ingress_values.yml
+    dest: '{{ nginx_ingress_value_file.path }}'
+
 - name: 'install nginx ingress'
   command: >-
-    helm install stable/nginx-ingress
-    --name nginx-ingress
+    helm upgrade --install nginx-ingress stable/nginx-ingress
+    --version 0.11.1
```

Comment:
This may require to run `helm repo update` at some point on the host, at least when upgrading.

Also, we may want to use `--repo`  to point at a specific repository (URL) rather than depending on repositories to be pre-configured on the host system.